src/Analyzer.Reports.UnitTests/TestCases.cs

```diff
@@ -795,6 +795,37 @@ public class TestCases
                     }
                 }
             },
+            new object[]
+            {
+                "Evaluations in only in referenced files",
```

Comment:
```suggestion
                "Evaluations is only in referenced files",
```
?

Reply:
I added a bit more detail to the module test cases to hopefully clear up